src/fujimn_int.cpp

```diff
@@ -192,6 +192,14 @@ namespace Exiv2 {
         { 32768, N_("Film simulation mode") }
     };
 
+    //! CropMode, tag 0x104d
+    extern const TagDetails fujiCropMode[] = {
+        { 0, N_("None")                          },
+        { 1, N_("None")                          },
```

Comment:
These don't match the translated values ExifTool thinks are right... Do you have more references?

Reply:
![screenshot_25](https://user-images.githubusercontent.com/529982/100078693-afa97a80-2e3b-11eb-84bd-b64b5e1ce25f.png)

Reply:
![screenshot_25](https://user-images.githubusercontent.com/529982/100078693-afa97a80-2e3b-11eb-84bd-b64b5e1ce25f.png)